app/css/icons/remove.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30"><path fill="#000" d="M 6.6191408,1.4609377 C 4.8997398,3.1803387 3.1803387,4.8997398 1.4609377,6.6191408 4.2545575,9.4127605 7.0481774,12.20638 9.8417972,15 7.0481774,17.79362 4.2545575,20.587239 1.4609377,23.380859 3.1803387,25.10026 4.8997398,26.819661 6.6191408,28.539062 9.4127605,25.745442 12.20638,22.951823 15,20.158203 c 2.79362,2.79362 5.587239,5.587239 8.380859,8.380859 1.719401,-1.719401 3.438802,-3.438802 5.158203,-5.158203 C 25.745442,20.587239 22.951823,17.79362 20.158203,15 22.951823,12.20638 25.745442,9.4127605 28.539062,6.6191408 26.819661,4.8997398 25.10026,3.1803387 23.380859,1.4609377 20.587239,4.2545575 17.79362,7.0481774 15,9.8417972 12.20638,7.0481774 9.4127605,4.2545575 6.6191408,1.4609377 Z" /></svg>
```

Comment:
Have you used svgo on these icons? It looks like they can be compressed even further.